Sources/Voice/OATTSCommandPlayerImpl.mm

```diff
@@ -69,11 +69,24 @@ - (void)playCommands:(OACommandBuilder *)builder
         [toSpeak appendString:utterance];
     }
     AVSpeechUtterance *utterance = [AVSpeechUtterance speechUtteranceWithString:toSpeak];
-    utterance.voice = [AVSpeechSynthesisVoice voiceWithLanguage:voiceProvider];
+    utterance.voice = [self voiceToUse];
     [utterance setRate:0.5f];
     [synthesizer speakUtterance:utterance];
 }
 
+- (AVSpeechSynthesisVoice *)voiceToUse {
+    NSArray<AVSpeechSynthesisVoice *> *allVoices = [AVSpeechSynthesisVoice speechVoices];
+    NSMutableArray<AVSpeechSynthesisVoice *> *allEnhancedVoicesForLanguage = [[NSMutableArray alloc] init];
+
+    for (AVSpeechSynthesisVoice *voice in allVoices) {
+        if (voice.quality == AVSpeechSynthesisVoiceQualityEnhanced && [voice.language hasPrefix:voiceProvider]) {
+            [allEnhancedVoicesForLanguage insertObject:voice atIndex:0];
+        }
+    }
+
+    return allEnhancedVoicesForLanguage.firstObject ? : [AVSpeechSynthesisVoice voiceWithLanguage:voiceProvider];
```

Comment:
The left value in the ternary operator is missing.

Reply:
Hi, have you made sure to [download the enhanced voices](https://www.aptus-slt.com/post/change-ios-voice) for your selected language?

Reply:
Hi, @ckuburlis. What I meant to say was that we do not generally use the ternary operator with the omitted left operand, but I guess it's a valid way to do it, so I will accept the pull request. Thank you for contributing.